suite-native/module-settings/src/slice.ts

```diff
@@ -18,6 +18,11 @@ const initialState: AppSettingsState = {
     isOnboardingFinished: false,
 };
 
+export const appSettingsPersistWhitelist: (keyof AppSettingsState)[] = [
```

Comment:
Please use `Array<keyof AppSettingsState>` for cases where type is more complicated, it looks better than parentheses